### PR TITLE
Rename MCS to Console in codebase

### PR DESCRIPTION
**Pull request by dvaldivia**
depends on #201

### CODE REVIEW

File: k8s/console/base/console-deployment.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
     matchLabels:
-      app: mcs
+      app: console
   template:
     metadata:
       labels:
-        app: mcs
+        app: console
     spec:
       serviceAccountName: m3-sa
```
Comment (Alevsk) [inline k8s/console/base/console-deployment.yaml:15]:
```suggestion
      serviceAccountName: console-sa
```

File: k8s/operator-console/base/console-deployment.yaml [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
     matchLabels:
-      app: mcs
+      app: console
   template:
     metadata:
       labels:
-        app: mcs
+        app: console
     spec:
       serviceAccountName: m3-sa
```
Comment (Alevsk) [inline k8s/operator-console/base/console-deployment.yaml:15]:
```suggestion
      serviceAccountName: console-sa
```

**Comment by dvaldivia** [comment]
@Alevsk all fixed

File: k8s/console/base/minio-operator.yaml [hunk 3]
```diff
@@ -1,14 +1,19 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: minio-operator
```
[the remark this answers is not in the document]
Comment (dvaldivia) [reply k8s/console/base/minio-operator.yaml]: this comes from operator repo

File: k8s/console/base/minio-operator.yaml [hunk 4]
```diff
@@ -90,6 +164,12 @@ spec:
           type: string
           jsonPath: ".status.currentState"
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: minio-operator
```
[the remark this answers is not in the document]
Comment (dvaldivia) [reply k8s/console/base/minio-operator.yaml]: this comes from operator repo

**Comment by cesnietor** [comment]
Error creating tenant:
```
2020-07-25T22:54:10.5460452Z E0725 22:54:10.545660       1 main-controller.go:329] error syncing 'default/minio-tenant-4': MCS_ACCESS_KEY not provided 
```

**Comment by dvaldivia** [comment]
@cesnietor those are a special case, since it's really an mc S3 Client :P

**Comment by cesnietor** [comment]
> @cesnietor those are a special case, since it's really an mc S3 Client :P

true on those only

**Comment by dvaldivia** [comment]
@cesnietor the other problem is addressed by https://github.com/minio/operator/pull/199

**Pull request merged**